food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```diff
@@ -57,28 +58,31 @@ public String postTruckReview(@AuthenticationPrincipal User user, @PathVariable
     }
 
     @Value
-    private static class CreateTruckParams {
+    public static class CreateTruckParams {
         @Nullable
         String truckName;
     }
 
+    @Secured({"ROLE_USER"})
```

Comment:
Once we get the "truck owner mode" implemented, we can add a ROLE_OWNER and use that instead